common/src/main/java/com/pg85/otg/generator/resource/OreGen.java

```diff
@@ -78,6 +79,12 @@ public String toString()
     @Override
     public void spawn(LocalWorld world, Random rand, boolean villageInChunk, int x, int z)
     {
+        
+        if (!bLoaded ) {
```

Comment:
Could put this in a method in Resource instead, no repeating code for each Gen class?